pkg/option/config.go

```diff
@@ -2022,6 +2038,36 @@ func (c *DaemonConfig) K8sAPIDiscoveryEnabled() bool {
 	return c.k8sEnableAPIDiscovery
 }
 
+func (c *DaemonConfig) validateIPAMClusterPoolV4CIDR() error {
+	if len(c.IPAMClusterPoolV4CIDR) != 0 {
+		for _, v4CIDR := range c.IPAMClusterPoolV4CIDR {
+			_, cidr, err := net.ParseCIDR(v4CIDR)
+			if err != nil {
+				return err
+			}
+			if cidr == nil {
+				return fmt.Errorf("IPAMClusterPoolV4CDIR ParseCIDR returned nil")
+			}
+		}
+	}
+	return nil
+}
+
+func (c *DaemonConfig) validateIPAMClusterPoolV6CIDR() error {
+	if len(c.IPAMClusterPoolV6CIDR) != 0 {
```

Comment:
You can remove this if statement, golang does not iterate over an empty size

Reply:
Ok will do.

Reply:
Done.